### ISSUE TITLE
Feature proposal: V-model to handle table refresh

### ISSUE THREAD

**Issue by cristijora** [ratiw/vuetable-2#189]
Currently in order to refresh vue-table you have to manually call
`this.$refs.tableRef.refresh()` 

This becomes quite cumbersome when you have multiple tables or when you have wrapper components for vue-table and have to handle this refs magic.

My proposal would be to have a v-model for refresh 
```html
<vue-table v-model="shouldRefresh">
</vue-table>
```
When changing shouldRefresh from false to true, the internal refresh method would be called and probably the model will be changed back to false.
This could get us rid of always handling refs which bring quite a few disadvantages

**Comment by ratiw** [comment]
@cristijora I think `v-model` wouldn't be a good choice, as it's a bit vague.

I would really prefer Vuetable to listen to an `refresh` event as in previous version to handle this kind of task, but it's kind of hard to tap into this as Vue 2 has shifted to centralize event hub model.

I left it as it is, so that anyone that wants to use Vuex could somehow wrap the call to `$refs.tableRef.refresh()` inside the action. I myself do not use Vuex, so I didn't continue on that.

In my own app, I use [`vue-events`](https://github.com/cklmercer/vue-events) for centralized event hub and listen to a trigger event somewhere that will call the `$refs.tableRef.refresh()` inside the event handler.

**Comment by lgullestrup** [comment]
I tend to agree using the $refs.tableRef.refresh() method. there is nothing to stop you from wrapping this into a helper method something like 

```
this.tblrefresh('tableref')

tblrefresh(tbl){
  //{{check make sure reference exists}}
  this.$refs[tbl].refrsh()
}
```

binding a value to the table make a lot of unnecessary over head as it would require 2 way binding, which goes against Vue's methodology. the .sync practice should only be used in certain situations. Vue tables being child component should listen to commands from the parent.
